build.sh

```diff
@@ -0,0 +1,76 @@
+clear
+
+# Add-on name
+echo "Enter Add-on name"
+echo -e "\e[32mExample: Give Funds\e[0m"
+read -p "" addon_name
+
+while [ -z "$addon_name" ]
+do
+  	read -p "Enter Add-on name: " addon_name
+done
+
+# Add-on description
+read -p "Add-on description: " addon_description
+
+# TextDomain
+echo "Enter add-on textdomain"
+echo -e "\e[32mExample: give-funds\e[0m"
+read -p "" addon_textdomain
+while [ -z "$addon_textdomain" ] || [[ ! "$addon_textdomain" =~ ^give([a-z-])*$ ]]
+do
+	echo -e "\e[32mExample: give-funds\e[0m"
+  	read -p "Enter valid add-on textdomain: " addon_textdomain
+done
+
+read -p "Build add-on? (y/n): " build_addon
+
+if [ "$build_addon" != y ] ; then
+	exit 1
+fi
+
+addon_constant=$( echo $( echo "$addon_name" | tr -s ' ' '_' ) | tr -cd "[:alnum:]_" )
+addon_namespace=$( echo "$addon_name" | tr -cd "[:alnum:]" )
+
+files=(
+	"src/Addon/Helpers/ActivationBanner.php"
+	"src/Addon/Helpers/Environment.php"
+	"src/Addon/Helpers/Language.php"
+	"src/Addon/Helpers/License.php"
+	"src/Addon/Helpers/Notices.php"
+	"src/Addon/Helpers/SettingsPage.php"
+	"src/Addon/Helpers/View.php"
+	"src/Addon/Activation.php"
+	"src/Addon/AddonServiceProvider.php"
+	"src/Addon/Assets.php"
+	"src/Addon/SettingsPage.php"
+	"src/Addon/resources/views/admin/notices/give-inactive.php"
+	"src/Addon/resources/views/admin/notices/give-version-error.php"
+	"give-addon-boilerplate.php"
+	"webpack.config.js"
+	"wp-textdomain.js"
+	"composer.json"
+)
```

Comment:
Having a fixed list here is begging for missed files in the future. Some of these root files are ok, but can we update to use a glob instead for the `src/` directory to grab all *.php files?

Reply:
I was thinking the same. Yes, we can do that.